src/com/cft/contactmerge/Contact.java

```diff
@@ -1,6 +1,6 @@
 package com.cft.contactmerge;
 
-public class Contact {
+public class Contact implements IContact {
```

Comment:
Just wanted to say I like how you are deciding to go with a polymorphic route here to separate the interface from the application.

Reply:
Thanks. We will definitely need it in a few places and so I figured we may as well start introducing it.